src/components/Profile/components/Identification/index.js

```diff
@@ -718,23 +726,38 @@ const Identification = ({ profile, myProf, network, createSnackbar }) => {
                     userProfile.Title !== '' &&
                     userProfile.PersonType === 'fac'
                       ? // If the user has a title
-                        hasNickName
-                        ? // If the user has a title and a nickname
-                          userProfile.Title +
-                          ' ' +
-                          userProfile.fullName +
-                          ' (' +
-                          userProfile.NickName +
-                          ')'
-                        : // If the user has a title and no nickname
-                          userProfile.Title + ' ' + userProfile.fullName
+                        userProfile.Title +
+                        ' ' +
+                        userProfile.fullName.split(' ')[0] +
+                        (hasNickname
+                          ? 
+                            ' (' +
+                            userProfile.NickName +
+                            ') ' 
+                            : ' ') +
+                        userProfile.fullName.split(' ')[2] +
+                        (hasMaidenName
+                          ? 
+                            ' (' +
+                            userProfile.MaidenName +
+                            ') ' 
+                            : '')
```

Comment:
We can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#expression_interpolation), a special string that you can embed expressions into, to clean this up a little bit:
```suggestion
                        `${userProfile.Title} ${userProfile.FirstName}${
                            hasNickname ? ` (${userProfile.NickName})` : ''
                        } ${userProfile.LastName}${
                            hasMaidenName ? ` (${userProfile.MaidenName})` : '')`
```

You could do the same thing for the case below as well (when there's no title).